.github/workflows/ci-gh.yml

```diff
@@ -1,7 +1,8 @@
 name: CI
 
 concurrency:
-  group: ${{ startsWith(github.ref_name, 'main') && format('unique-{0}', github.run_id) || format('ci-build-and-test-on-{0}-from-{1}', github.event_name, github.ref_name) }}
+  #group: ${{ startsWith(github.ref_name, 'main') && format('unique-{0}', github.run_id) || format('ci-build-and-test-on-{0}-from-{1}', github.event_name, github.ref_name) }}
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
TODO: revert this logic (I plan to touch it in a separate PR) once this action can be canceled (it currently hangs when requesting a possibly nonexistent P100 runner, which seems removed very recently? I've seen that when being stuck in the runner queue the workflow cancellation has no effect. cc @ajschmidt8 for vis)
https://github.com/NVIDIA/cuda-python/actions/runs/12523539614/job/34933264754

Reply:
It seems the runner support team is out. I updated the group name in commit 2482fe7 to avoid the hanging job from blocking further CI triggers.

Reply:
> I plan to touch it in a separate PR

I decided to do it in this PR and nuke the ci-gh file, see commit 131e7b7b601239b73839d642f7765e13d410dd59